NRLMMD-GEOIPS/geoips
# Optimize/773 conda installation

# Reviewer Checklist

PR author: Please ensure you meet all of the below requirements, and check boxes appropriately.

Reviewers: Please confirm all required testing/documentation has been completed prior to approving.

**Remove lines that are not applicable, explain if you select "NO REQUIRED"**

* [ ] NO REQUIRED ***existing tests*** (Changes only in documentation)
* [ ] NO REQUIRED ***unit tests*** (Changes only in documentation)
* [ ] NO REQUIRED ***integration tests*** (Changes only in documentation)
* [ ] NO REQUIRED ***release notes*** (Minor change in the installation)
* [ ] NO REQUIRED ***updates to other repos*** (Changes only in documentation)


# Related Issues
fixes NRLMMD-GEOIPS/geoips#773

# Testing Instructions

The following would validate the proposed changes. 

- wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
- accept the license and confirm the location. Enter no for other prompts.
- do not close your terminal instead run `source ~/.bashrc` 
- you should see base as your environment
- run mamba create -y -n geoips -c conda-forge python=3.10 gcc gxx openblas git
- conda activate geoips 

# Summary

Description

Currently, when installing the Conda package manager, users are required to close and reopen the current shell for the initial configurations to take effect. This PR addresses the need to apply Conda's initial configuration without closing the terminal window.

![hello](https://github.com/user-attachments/assets/cd6ddd5f-9ab6-4af9-8843-085bc6ac489c)



This is particularly beneficial for users installing GeoIPS on servers, as it avoids the need to disconnect and reconnect, while also maintaining the current session seamlessly.




# Output
